### DISCUSSION TITLE
Question on the typec port of the lite build

### DISCUSSION THREAD

**Discussion by Stevey233** [osy/XG_Mobile_Station#15] [Hardware] [answered]
I am trying to build my own lite dock,but I dont fully understand what's the c port on it for.Does I need to connect it to my PC or external monitor?Would it work if I just leave the C port unplugged?
Please forgive my poor english I am not a native speaker and I would be glad if you can give some explanation on the function of the c port on the lite build

**Comment by osy** [comment, marked as answer]
On the Ally you have one usb c port. How do you charge your device? How do you connect a USB keyboard and mouse while XGM is connected?
